File: easy/6_is_leap_year/is_leap_year.py
# Check leap year

def is_leap_year(year):
    if year % 4 == 0 and year % 100 != 0 or year % 400 == 0:
        return True
    return False

print(is_leap_year(2000))  # True
print(is_leap_year(2001))  # False
print(is_leap_year(2004))  # True
print(is_leap_year(2100))  # False

File: easy/6_is_leap_year/is_leap_year.js
// Check leap year

function is_leap_year(year) {
    if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
        return true;
    }
    return false;
}

console.log(is_leap_year(2000));  // true
console.log(is_leap_year(2001));  // false
console.log(is_leap_year(2004));  // true
console.log(is_leap_year(2100));  // false

module.exports = is_leap_year;